src/chains/ethereum_1/ethChainState.ts

```diff
@@ -131,26 +131,37 @@ export class EthereumChainState {
     }
   }
 
+  getTransactionHash(signedTransaction: string) {
```

Comment:
This function sends the transaction and returns a receipt right? If so this is a VERY misleading function name.
First, a function named 'get...' should never mutate data - very unexpected - a 'get' should return something not normally do something.
Second, this function name should be clear why it exists and what it does - something like sendTransactionWithoutWaitingForConfirm() - or something similar